website/docs/r/cloud_backup_schedule.html.markdown

```diff
@@ -0,0 +1,346 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cloud_backup_schedule"
+sidebar_current: "docs-mongodbatlas-resource-cloud-backup-schedule"
+description: |-
+    Provides a Cloud Backup Schedule resource.
+---
+
+# mongodbatlas_cloud_backup_schedule
+
+`mongodbatlas_cloud_backup_schedule` provides a cloud backup schedule resource. The resource lets you create, read, update and delete policies items.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage - Create a Cluster with no policies
```

Comment:
Change line 15 to:
 ## Example Usage - Create a Cluster with Cloud Backup Enabled but No Policy Items

Add a blank line and then the following:

You can enable cloud_backup in the Cluster resource and then use the cloud_backup_schedule resource with no policy items to remove the default policy that Atlas creates when you enable Cloud Backup.  This allows you to then create a policy when you are ready to via Terraform.